packages/manager/apps/telecom/src/app/telecom/pack/slots/emailPro/emailPro.controller.js

```diff
@@ -0,0 +1,38 @@
+import map from 'lodash/map';
+
+export default /* @ngInject */ class PackEmailProCtrl {
```

Comment:
on `class` `/* @ngInject */` should be added line before `constructor` (example: https://github.com/ovh/manager/blob/master/packages/manager/apps/telecom/src/app/telecom/telephony/telephony.controller.js#L3-L5)